server/src/main/java/com/flipkart/varadhi/CoreServices.java

```diff
@@ -46,8 +46,8 @@ public CoreServices(ServerConfiguration configuration) {
     }
 
 
-    public Tracer getTracer(String instrumentationScope, String version) {
-        return this.observabilityStack.getOpenTelemetry().getTracer(instrumentationScope, version);
+    public Tracer getTracer(String instrumentationScope) {
```

Comment:
Version is the version associated with a instrumentation scope for tracing. Since in the current context, entire Varadhi is the instrumentation scope, it would be the version of Varadhi in traces. 
I am not sure if we have Varadhi version view yet and also which needs to be differentiated in Varadhi. 
https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope